### PR TITLE
meal-sharing NodejsWeek2

### PR DESCRIPTION
**Pull request by dmytriigithub**

### CODE REVIEW

File: src/backend/api/meals.js [hunk 1]
```diff
@@ -1,15 +1,67 @@
...
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
-  } catch (error) {
+    const allMeals = await knex("Meal").select("*")
+    if(!allMeals.length){
+      response.status(404).json({ error: 'That shit doesn’t exist.' });
+    }
+    response.json(allMeals);
```
Comment (Jane-API-SG) [inline src/backend/api/meals.js:11]: This should be moved to the try part.

File: src/backend/api/meals.js [hunk 2]
```diff
@@ -1,15 +1,67 @@
...
+  } 
+  catch (error) {
     throw error;
   }
```
Comment (Jane-API-SG) [inline src/backend/api/meals.js]: Is this catch statement needed?

File: src/backend/api/meals.js [hunk 3]
```diff
@@ -1,15 +1,67 @@
...
+    const id = parseInt(request.params.id);
+    const idMeal = await knex("Meal").select("*").where("id", '=', id);
+    if(!idMeal.length){
+      response.status(404).json({ error: 'That shit doesn’t exist' });
+    }
+    response.json(idMeal);
+  } 
+  catch (error) {
+    throw error;
+  }
```
Comment (Jane-API-SG) [inline src/backend/api/meals.js]: Try handling the errors in the catch statement.

**Review by Jane-API-SG — APPROVED** [review]

**Review by Jane-API-SG — APPROVED** [review]
Looks good :ok_hand:

File: src/backend/api/meals.js (hunk 3 above)
Comment (dmytriigithub) [reply src/backend/api/meals.js]: Thanks for review and feedback after changing

**Pull request merged**